classifiers/image/CLIPZeroShotClassifier/tests/test_clipzeroshotclassifier.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import clip
+import numpy as np
+from PIL import Image
+
+from .. import CLIPZeroShotClassifier
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+def test_clipzeroshotclassifier():
+    dog_image = os.path.join(cur_dir, '../imgs/dog.jpg')
```

Comment:
I suggest to put it in the `tests` folder
```suggestion
    dog_image = os.path.join(cur_dir, 'imgs/dog.jpg')
```